.gitmodules

```diff
@@ -0,0 +1,9 @@
+[submodule "notifier-dev"]
+	path = notifier-dev
+	url = git@github.com:rsksmart/rif-marketplace-notifier.git
+[submodule "rns-dev"]
+	path = rns-dev
+	url = git@github.com:rsksmart/rif-marketplace-rns-dev-env.git
```

Comment:
why don't we just use the nfts repo?

Reply:
Wondered about that myself... I only used the repos that were already here.